src/core/issues-notes.js

```diff
@@ -239,12 +241,24 @@ function isLight(rgb) {
  * @param {string} title
  * @param {string} repoURL
  */
-function createLabelsGroup(labels, title, repoURL) {
+function createLabelsGroup(conf, labels, title, repoURL) {
   const labelsGroup = labels.map(label => createLabel(label, repoURL));
+  let labelName = labels.map(label => label.name);
```

Comment:
This should be `const`, for the final string, create a new variable. That avoids doing a type change. 

```suggestion
  const labelNames = labels.map(label => label.name);
```